Packet++/src/GtpLayer.cpp

```diff
@@ -155,522 +155,521 @@ GtpV1Layer::GtpExtension GtpV1Layer::GtpExtension::createGtpExtension(uint8_t* d
+	}
 }
 
 size_t GtpV1Layer::getHeaderLen()
 {
-    gtpv1_header* header = getHeader();
-    if (header == NULL)
-    {
-        return 0;
-    }
-
-    size_t res = sizeof(gtpv1_header);
-
-    if (header->messageType != PCPP_GTP_V1_GPDU_MESSAGE_TYPE)
-    {
-        size_t msgLen = be16toh(header->messageLength);
-        res += (msgLen > m_DataLen - sizeof(gtpv1_header) ? m_DataLen - sizeof(gtpv1_header) : msgLen);
-    }
-    else
-    {
-        gtpv1_header_extra* headerExtra = getHeaderExtra();
-        if (headerExtra != NULL && (header->extensionHeaderFlag == 1 || header->sequenceNumberFlag == 1 || header->npduNumberFlag == 1))
-        {
-               res += sizeof(gtpv1_header_extra);
-            GtpExtension nextExt = getNextExtension();
-            while (!nextExt.isNull())
-            {
-                res += nextExt.getTotalLength();
-                nextExt = nextExt.getNextExtension();
-            }
-        }
-    }
-
-    return res;
+	gtpv1_header *header = getHeader();
+	if (header == NULL)
+	{
+		return 0;
+	}
+
+	size_t res = sizeof(gtpv1_header);
+
+	if (header->messageType != PCPP_GTP_V1_GPDU_MESSAGE_TYPE)
+	{
+		size_t msgLen = be16toh(header->messageLength);
+		res += (msgLen > m_DataLen - sizeof(gtpv1_header) ? m_DataLen - sizeof(gtpv1_header) : msgLen);
+	}
+	else
+	{
+		gtpv1_header_extra *headerExtra = getHeaderExtra();
+		if (headerExtra != NULL && (header->extensionHeaderFlag == 1 || header->sequenceNumberFlag == 1 || header->npduNumberFlag == 1))
+		{
+			res += sizeof(gtpv1_header_extra);
+			GtpExtension nextExt = getNextExtension();
+			while (!nextExt.isNull())
+			{
+				res += nextExt.getTotalLength();
+				nextExt = nextExt.getNextExtension();
+			}
+		}
+	}
+
+	return res;
 }
 
 std::string GtpV1Layer::toString()
 {
-    std::string res = "GTP v1 Layer";
+	std::string res = "GTP v1 Layer";
 
-    gtpv1_header* header = getHeader();
-    if (header != NULL)
-    {
-   		std::stringstream teidStream;
+	gtpv1_header *header = getHeader();
+	if (header != NULL)
+	{
+		std::stringstream teidStream;
 		teidStream << be32toh(header->teid);
 
-        std::string gtpu_gtpc;
-        if (header->messageType == PCPP_GTP_V1_GPDU_MESSAGE_TYPE)
-        {
-            gtpu_gtpc = "GTP-U message";
-        }
-        else
-        {
-            gtpu_gtpc = "GTP-C message: " + getMessageTypeAsString();
-        }
-        
-        res += ", " + gtpu_gtpc + ", TEID: " + teidStream.str();
-    }
+		std::string gtpu_gtpc;
+		if (header->messageType == PCPP_GTP_V1_GPDU_MESSAGE_TYPE)
+		{
+			gtpu_gtpc = "GTP-U message";
+		}
+		else
+		{
+			gtpu_gtpc = "GTP-C message: " + getMessageTypeAsString();
+		}
+
+		res += ", " + gtpu_gtpc + ", TEID: " + teidStream.str();
+	}
 
-    return res;
+	return res;
 }
 
 void GtpV1Layer::computeCalculateFields()
 {
-    gtpv1_header* hdr = getHeader();
-    if (hdr == NULL)
-    {
-        return;
-    }
+	gtpv1_header *hdr = getHeader();
```

Comment:
please move the `*` back to its original place. This is the convention in the rest of the project